src/rust/Cargo.lock

```diff
@@ -200,6 +201,11 @@ name = "keccak"
 version = "0.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
+[[package]]
+name = "lazy_static"
```

Comment:
I was convinced to have seen lazy static before somewhere in the firmware code but maybe I was dreaming.

Reply:
I had it in the PoC PR about this, which I closed.